docs/index.md

```diff
@@ -84,6 +84,8 @@ You'll need to set up new Shopify API credentials to connect Theme Kit to your s
 
 1. In the store's Shopify admin, click **Apps**.
 1. Near the bottom of the page, click on **Manage private apps**.
+1. If you see a notice that "Private app development is disabled", click "Enable
```

Comment:
Our style guide requires "ifs" to have a corresponding "then"

```suggestion
1. If you see a notice that "Private app development is disabled", then click "Enable
```

Reply:
Also seems like maybe an IDE linter added a line-break here? Doesn't affect the HTML output but I like keeping list items on one line if possible for readability